### PR TITLE
Add button Logout and information response status

### PR DESCRIPTION
**Pull request by ozimas**

### CODE REVIEW

File: static/js/script.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function checkLoginState() {
 
 function statusChangeCallback(response){
     console.log(response);
```
Comment (tss89) [inline static/js/script.js]: remove console.log

File: static/js/script.js [hunk 2]
```diff
@@ -37,4 +38,17 @@ function statusChangeCallback(response){
             console.log(data);
```
Comment (tss89) [inline static/js/script.js]: remove console.log

File: static/js/script.js [hunk 3]
```diff
@@ -37,4 +38,17 @@ function statusChangeCallback(response){
...
         }
     });
+
+    changeStatus(response.status);
+}
+
+function logoutFacebook(){
+    FB.logout(function(response) {
+        console.log(response);
```
Comment (tss89) [inline static/js/script.js]: remove console.log

File: static/js/script.js (hunk 1 above)
Comment (ozimas) [reply static/js/script.js]: Done.

File: static/js/script.js (hunk 2 above)
Comment (ozimas) [reply static/js/script.js]: Done.

File: static/js/script.js (hunk 3 above)
Comment (ozimas) [reply static/js/script.js]: Done.

**Pull request merged**